extensions/cornerstone/src/utils/measurementServiceMappings/Angle.ts

```diff
@@ -76,11 +76,12 @@ const Angle = {
       data: data.cachedStats,
       type: getValueTypeFromToolType(toolName),
       getReport,
+      nonAcquisition: referencedImageId == null,
```

Comment:
can we just insert the referencedImageId in here, it might be useful in future

Reply:
so for nonAcquisition you can check its presence

Reply:
Sure